src/components/ErrorComponent/ErrorComponentContent.ts

```diff
@@ -0,0 +1,26 @@
+import type { Picture } from '../../model/Picture';
+import type { ButtonProps } from '../../ui-kit/Button/ButtonProps';
+
+export type ErrorBlockDef = 'image' | 'number';
+
+export interface ErrorBlockContent {
+  label: ErrorBlockDef;
+  /** @title Изображение */
+  image?: Picture;
+  /** @title Число */
+  numberValue?: number;
+}
```

Comment:
не правильный интерфейс. Как я уже говорил, нужно использовать механизм который используется в аккардионе. Мы должны получить на выходе `anyOf`, который в форме будет интерпретироваться как дропдаун. Также нарушен конвеншен по использованию префикса `*Content`.

```suggestion
/**
 * @title Изображение
 */
export type ErrorImageDef =  {
    /** @default Image */
    errorContentType: 'Image';
    image?: Picture;
};

/**
 * @title Код ошибки
 */
export type ErrorCodeDef =  {
    /** @default Code */
    errorContentType: 'Code';
    code?: number;
};

export type ErrorContentDef = EmptyOption | ErrorImageDef | ErrorCodeDef;
```

Также требуется в КР в ui:schema скрыть поле errorContentType